component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/emailotp/util/EmailOTPAuthErrorConstants.java

```diff
@@ -0,0 +1,51 @@
+package org.wso2.carbon.identity.authenticator.emailotp.util;
+
+public class EmailOTPAuthErrorConstants {
+
+    /**
+     * Authentication error constants of EmailOTP Authenticator.
+     */
+    public enum ErrorMessages {
+
+        // Identifier related Error codes.
+        EMPTY_USERNAME("BAS-60002", "Username is empty."),
```

Comment:
What does `BAS` stand for?

Reply:
This was an error code taken from Basic authenticator. I replaced it with a unique code to email OTP in a later commit.